iml-gui/crate/src/lib.rs

```diff
@@ -54,7 +54,7 @@ const MAX_SIDE_PERCENTAGE: f32 = 35_f32;
 /// help url becomes `https://localhost:8443/help/docs/Graphical_User_Interface_9_0.html`
 const CTX_HELP: &str = "help/docs/Graphical_User_Interface_9_0.html";
 
-pub fn extract_api(s: &str) -> Option<&str> {
+pub fn extract_id(s: &str) -> Option<&str> {
```

Comment:
Should this be in an api crate? I feel like this might get used in a few different places.

Reply:
It already is in a few places, and yeah we should look at a way to extract common api bits that are shared between the front-end / backend.